lib/utils.js

```diff
@@ -22,12 +22,10 @@ const XCTEST_LOGS_CACHE_FOLDER_PREFIX = 'com.apple.dt.XCTest';
 
 const CONSTRAINTS = {
   webkitDebugProxyPort: {
-    isNumber: true,
-    presence: true
+    isNumber: true
   },
   wdaLocalPort: {
-    isNumber: true,
-    presence: true
+    isNumber: true
   },
 };
 
```

Comment:
same comment as for https://github.com/appium/appium-android-driver/pull/693/files#r643668254